docs/sparrow.md

```diff
@@ -16,7 +16,6 @@ The check results are exposed via an API.
 
 ### SEE ALSO
 
-* [sparrow completion](sparrow_completion.md)	 - Generate the autocompletion script for the specified shell
-* [sparrow gen-docs](sparrow_gen-docs.md)	 - Generate markdown documentation
 * [sparrow run](sparrow_run.md)	 - Run sparrow
 
+###### Auto generated by spf13/cobra on 10-Jan-2024
```

Comment:
Disable this auto gen tag since the pre-commit hook would always fail once a day.